packages/core/ts/MobileCore.ts

```diff
@@ -115,7 +115,7 @@ const MobileCore: IMobileCore = {
    * more verbose than the current {@link LogLevel} set from {@link #setLogLevel(LogLevel)}
    * then the message is not printed.
    *
-   * @param mode the {@link LogLevel} used to print the message
+   * @param logLevel the {@link LogLevel} used to print the message
    * @param tag used to identify the source of the log message
    * @param message the message to log
    */
```

Comment:
In Set the Adobe Mobile Privacy status below
* @param {PrivacyStatus} PrivacyStatus to be set to the SDK